src/PortingAssistant.Client.Analysis/AnalysisHandler.cs

```diff
@@ -393,6 +393,9 @@ private List<ProjectResult> AnalyzeActions(List<string> projects, string targetF
                 var analyzer = analyzers.Find((a) => a.ProjectResult?.ProjectFilePath != null &&
                     a.ProjectResult.ProjectFilePath.Equals(project));
 
+                var projectFeatureType = analysisActions.Find((a) => a.ProjectFile != null &&
+                    a.ProjectFile.Equals(project))?.FeatureType.ToString();
```

Comment:
please address the error with object reference

Reply:
Will probably require a CTA release to resolve this- FeatureType is a new property on the ProjectResult object which is defined in CTA.

Reply:
Yep this is pending CTA update